Makefile

```diff
@@ -41,6 +43,12 @@ fmt:
 vet:
 	go vet ./...
 
+unit:
+	go test -cover -tags unit ./...
+
+mockgen:
+	go generate ./...
```

Comment:
should we add some code here that checks for `mockgen` and installs it if it isn't found?  similar to what the `generate` make target does

Reply:
Yeah that would be useful. I added a check and `go get` right before the generate call, although I didn't make it a separate target like the `controller-gen` is.